src/replaceSecrets.index.ts

```diff
@@ -0,0 +1,16 @@
+import { writeFile } from 'fs';
+
+const targetPath = './src/environments/environment.ts';
+
+const envConfigFile = `export const environment = {
+  production: false,
+  TravelTrackAPI: '${process.env?.['TRAVELTRACK_API_BASE_URL']}',
+  TravelTrackAPIKey: '${process.env?.['TRAVELTRACK_API_KEY']}',
+  GooglePlacesAPIKey: '${process.env?.['GOOGLE_PLACES_API_KEY']}'
```

Comment:
I recommend having something in your readme indicating that if you want to run this locally, you will need to have these environment variables set. You should give an example of a URL as well so people know if it should have a trailing slash or not.